onetimepass/otp.py

```diff
@@ -8,9 +8,19 @@
 
 from onetimepass import algorithm
 from onetimepass import settings
+from onetimepass.db import BaseDB
+from onetimepass.db import DatabaseSchema
+from onetimepass.db import DBDoesNotExist
 from onetimepass.db import JSONEncryptedDB
 
 
+def get_db_data(db: BaseDB) -> DatabaseSchema:
+    try:
+        return db.read()
+    except DBDoesNotExist:
+        raise click.UsageError("Database does not exist. Try to initialize it first")
```

Comment:
Let's guide user by explicitly suggesting to use `otp init` (although we should probably abstract `otp` binary name somehow).